test/tests/functional/pbs_suspend_resume_accounting.py

```diff
@@ -221,8 +218,11 @@ def test_resc_released_susp_resume(self):
         self.server.sigjob(jobid=jid, signal="suspend")
 
         # check for both ncpus and mem are released
-        resc_released = 'resources_released=(%s:ncpus=1:mem=524288kb)' % \
-                        self.server.shortname
+        resc_released = 'resources_released=(%s:ncpus=1:mem=524288kb)'
+        if self.mom.is_cpuset_mom():
+            resc_released = resc_released % self.server.status(NODE)[1]['id']
```

Comment:
Instead of using the first vnode here, can we do either of this

- Submit a job specifying the vnode and use it while verifying both ncpus and mem are released.
**(or)**
- Take the node name from exec_vnode attribute of the job while verifying

Reply:
Thanks @Mahalty. I opted for the second one. Please review the latest commit.